### PR TITLE
Update to latest ace-builds release

### PR DESCRIPTION
**Pull request by dfreeman**
Also fixing usage of a deprecated `ember-cli-page-object` API, since I'm going to release this as 2.0 anyway.

/cc @amyrlam @jamesarosen

### CODE REVIEW

File: addon-test-support/components/ember-ace/autocomplete.js [hunk 1]
```diff
@@ -72,14 +64,9 @@ export default {
...
-    get() {
-      for (let i = 0, len = this.suggestions().count; i < len; i++) {
-        if (this.suggestions(i).selected) {
-          return i;
-        }
-      }
-      return -1;
-    }
+    get: editorInteraction((editor) => {
+      return editor.completer.popup.getRow();
```
Comment (jamesarosen) [inline addon-test-support/components/ember-ace/autocomplete.js:68]: 👍

File: addon-test-support/components/ember-ace/autocomplete.js [hunk 2]
```diff
@@ -88,20 +75,17 @@ export default {
   focusedSuggestion: {
     isDescriptor: true,
     get() {
-      return this.suggestions(this.focusedIndex);
+      return this.suggestions.objectAt(this.focusedIndex);
```
Comment (jamesarosen) [inline addon-test-support/components/ember-ace/autocomplete.js:78]: 👍

File: tests/integration/components/ember-ace-test.js [hunk 3]
```diff
@@ -154,41 +154,41 @@ module('Integration | Component | ember ace', function(hooks) {
...
+    assert.equal(this.component.frontMarkers.objectAt(1).type, 'ember-ace-info');
+    assert.equal(this.component.frontMarkers.objectAt(1).segmentCount, 1);
   });
 
   test('basic autocomplete', async function(assert) {
     this.set('suggestCompletions', (editor, session, position, prefix) => {
       return [
         { value: `${prefix}abc`, caption: 'lhs', meta: 'rhs' },
-        { value: `${prefix}def`, caption: 'lhs2', meta: 'rhs2' },
+        { value: `${prefix}def`, caption: 'foo', meta: 'bar' },
```
Comment (jamesarosen) [inline tests/integration/components/ember-ace-test.js:176]: Fair enough

**Review by jamesarosen — APPROVED** [review]
+1. Nice work :)

File: tests/integration/components/ember-ace-test.js (hunk 3 above)
Comment (dfreeman) [reply tests/integration/components/ember-ace-test.js:176]: Looking at the way the sorting works within Ace now, it doesn't seem like there's a way to restore the old "just show it in the order I supplied it" behavior, so I figured I might as well embrace it in the tests ¯\\\_(ツ)_/¯

**Pull request merged**